.github/workflows/build/Dockerfile.ubuntu-20-04

```diff
@@ -102,7 +102,7 @@ RUN mv /usr/lib/ursa/* /usr/lib && rm -rf /usr/lib/ursa
 # 	apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys CE7709D068DB5E88 && \
 RUN echo "deb https://repo.sovrin.org/sdk/deb bionic master" >> /etc/apt/sources.list && \
     apt-get update -y && apt-get install -y \ 
-    libindy=1.13.0~1420
+    libindy=1.15.0~1625
```

Comment:
Should the `libindy` version in `.github/workflows/build/Dockerfile.ubuntu-20-04` and `ci/ubuntu.dockerfile` match?

`.github/workflows/build/Dockerfile.ubuntu-20-04`:
```
RUN echo "deb https://repo.sovrin.org/sdk/deb bionic master" >> /etc/apt/sources.list && \
    apt-get update -y && apt-get install -y \ 
    libindy=1.15.0~1625
```

`ci/ubuntu.dockerfile`:
```
RUN apt-get update -y && apt-get install -y \
    python3-nacl \
    libindy=1.15.0~1625-xenial \
```

Reply:
Related to; https://github.com/hyperledger/indy-plenum/pull/1533#discussion_r613405407

Reply:
@adenishchenko, This PR should be updated to match; https://github.com/hyperledger/indy-plenum/pull/1533/files